### DISCUSSION TITLE
[Request] Vikunja

### DISCUSSION THREAD

**Discussion by timdonovanuk** [tteck/Proxmox#678] [Ideas]
Starting be quite a popular open source Todoist replacement. Just had another big release.

https://vikunja.io
https://vikunja.io/docs/install-backend/#debian-packages
https://vikunja.io/docs/install-backend/#install-from-binary
https://vikunja.io/docs/install-frontend/

Thanks!

**Comment by wisie** [comment]
Another vote :)

**Reply by Koliera** [reply]
If you install docker/portainer lcx then this is simple to follow - https://www.youtube.com/watch?v=DqyqDWpEvKI

**Comment by Baymax2909** [comment]
I can only support this! Would be nice and helpful to get a script with Vikunja :)

**Comment by Gangstrocity** [comment]
Adding my support for this one as well.

**Comment by nosar77** [comment]
I would love this as well or leantime!

**Comment by vednolacni** [comment]
Any update on this?

**Comment by ptrsmk** [comment]
This would be a great addition!
